pkg/workspace.go

```diff
@@ -290,6 +290,9 @@ func (c *Client) addRuntimeFieldsToWorkspaceTemplate(t wfv1.Template, workspace
 		return nil, errors.New("unable to type check statefulset manifest")
 	}
 	extraContainer := generateNodeCaptureContainer(workspace, config)
+	resourcesRaw := extraContainer["resources"]
```

Comment:
Suggestion to move the code from `generateNodeCaptureContainer` to this method:
![image](https://user-images.githubusercontent.com/1211823/101969142-af4d0580-3bd7-11eb-8172-7f7b87a29b72.png)